packages/esm-service-queues-app/src/queue-table/default-queue-table.component.tsx

```diff
@@ -133,7 +133,7 @@ function DefaultQueueTable() {
               queueUuid={null}
               statusUuid={null}
               ExpandedRow={QueueTableExpandedRow}
-              tableFilter={[
+              tableFilters={[
```

Comment:
I've encountered a similar scenario in the ward app and decided that it's better to just type `tableFilters` as `ReactNode` (instead of `ReactNode[]`) and then do 

```
<>
  <QueueDropdownFilter />
  <StatusDropdownFilter />
  ...
</>
```

Then we don't have do deal with keys.

Reply:
Should we switch to that here instead?

Reply:
sure